app/views/categories/index.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class='categories-container'>
+  <div class='sign-up-details category-header'>
+    <%= image_tag 'hamburger.png', class: 'hamburger' %>
+    <h2>Categories</h2>
+    <%= image_tag 'search.png', class: 'search' %>
+  </div>
+  <div class='card-main-container'>
+    <% if @categories.empty? %>
+      <h3 class='center-text'>Add a category</h3>
+    <% else %>
+      <% @categories.each_with_index do |category, index| %>
+        <a href='/categories/<%= category.id %>/details'>
+          <div class='card-container'>
+            <% if category.present? %>
+              <div class='com-icon-container'>
+                <img class='company-icon' src="<%= category.icon %>" alt='category-icon'></img>
+              </div>
+              <div class='company-date'>
+                <p><%= category.name %></p>
+                <p class='date'><%= category.created_at.strftime('%e %b %Y') %></p>
+              </div>
+              <p>$ <%= @total_amount[index] %></p>
+            <% else %>
+              <p>Add a category</p>
+            <% end %>
+          </div>
+        </a>
+      <% end %>  
+    <% end %>   
+  </div>
+  <a href="<%= new_category_path %>">
+    <%= image_tag 'add.png', class: 'add-item' %>
+  </a>
```

Comment:
- [ ] I must be honest I love your design and your work, but you have to keep to design guidelines, your add button tho beautiful does not look like what we have in the design guide, kindly fix that
your design
![image](https://github.com/skabeo/23-books/assets/66097371/83da04c4-8b50-4471-a52a-b07c627a72d0)
design guide
![Screenshot from 2023-04-27 19-57-40](https://github.com/skabeo/23-books/assets/66097371/6eb38989-3b02-4e22-af6e-972a91705030)